addon/components/initials-avatar.js

```diff
@@ -7,7 +7,21 @@ export default Ember.Component.extend({
   colorIndex: 1,
   maxColorIndex: 1,
 
-  hasImage: Ember.computed.notEmpty('image'),
+  checkImage: function() {
+    var self = this;
+    $.ajax(self.get('image'), {
+      success: function(resp) {
+        self.set('hasImage', true);
+      },
+    });
```

Comment:
Yeah you should use the Image API to do that and the onLoad event. it seems weird to trigger an ajax call from inside a component